src/main/java/pl/akademia/api/BikeService.java

```diff
@@ -37,4 +39,22 @@ public int deleteBikeById(Long id) {
   public void switchPrice(BigDecimal value) {
     bikeRepository.switchPrice(value);
   }
+
+  @Transactional
```

Comment:
Czy adnotacja @Transactional jest tu wymagana? Jeżeli dobrze rozumiem jej funkcjonalność to generalnie powinna być użyta w przypadku gdy modyfikujemy dane w DB. W tym przypadku tylko je odczytujemy.